src/components/wrapper/wrapper.test.js

```diff
@@ -32,9 +33,15 @@ describe('Wrapper', () => {
     expect(container.length).toBe(0);
   });
 
-  it('does not display the settings modal when displayGlobalToolbar is false', () => {
-    const wrapper = setup.mount(Wrapper, mockPropsNoGlobalToolbar);
-    const container = wrapper.find('.pipeline-settings-modal');
+  it('does not perform any routing when displayGlobalToolbar is false', () => {
+    const wrapper = setup.mount(
+      <MemoryRouter initialEntries={['/experiment-tracking']}>
+        <Wrapper />
+      </MemoryRouter>,
+      mockPropsNoGlobalToolbar
+    );
+
+    const container = wrapper.find('.pipeline-ui--experiment-tracking');
```

Comment:
Thanks for adding this test - this is definitely structured in a much better way.

I've just had a play around with this and tried testing this using `mockProps` instead of `mockPropsNoGlobalToolbar` but unfortunately this still passes, which indicates that there is something missing from this setup. 

This could either be that we are testing with the wrong css class, or something to do with the setup - worth to do a bit of digging around to find out and set up the right test.

Reply:
I'm struggling on this one. I don't see a good way to test this.

Since we'll only render the `FlowChartWrapper` when we use `mockPropsNoGlobalToolbar`, we won't have access to any routing, so actually my use of `MemoryRouter` doesn't make sense here.

And since we don't have access to that routing or even `react-router` at this stage, I don't see a way to mock navigation to one of the routes using any of the methods you [normally would use](https://v5.reactrouter.com/web/guides/testing) when routing is present on the page or in the component.

If you have a better idea, do please share it, because I can't see it.